docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+
+Pull request titles should begin with a descriptive prefix (e.g., "[ENH] Implement check for presence of a session ID column"):
+
+- `[ENH]:` Feature improvements or additions
+- `[REF]:` Refactoring existing code
+- `[TST]`: Updating or adding a test
+- `[CI]`: Automation-related changes
+- `[MNT]`: General maintenance not covered by `[REF]`, `[TST]`, or `[CI]`
+- `[INF]`: Software or graph infrastructure-related changes
+- `[FIX]`: Bug fixes
+- `[MODEL]`: Updates or changes related to the Neurobagel data model
+- `[DOC]`: Documentation-only changes to a code repo (READMEs, within-code documentation, etc.) 
+    - Exception: changes to the [`documentation`](https://github.com/neurobagel/documentation) repo should use one of the below PR prefixes instead of `[DOC]`
+
+If you are an external contributor, once your pull request is opened,
+we recommend that you [request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from a [Neurobagel core maintainer](http://127.0.0.1:8000/contributing/team/#developers) to ensure it gets reviewed in a timely manner.
+
+### Pull requests to the [`documentation`](https://github.com/neurobagel/documentation) repo
+In PRs to the [Neurobagel documentation](https://github.com/neurobagel/documentation), using the `[DOC]` title prefix is discouraged as it is too broad. 
+Instead, for documentation content changes, the following prefixes can be used to specify the nature of the change:
+
+- `[ENH]`: Updating or adding new documentation
+- `[REF]`: Simplifying or restructuring documentation (i.e., pages, sections, paragraphs)
+- `[FIX]`: Fixing errors in the documentation
+
+## Pull request reviews
+A core maintainer will review each PR and may provide comments or suggestions for the PR author to address.
+
+Neurobagel PR reviews may use the following emoji signifiers:
+
+:cook:: Ready to merge or approved without suggestions
+
+:cherries:: Some optional/suggested changes that could be nice to have but are not required to merge
+
+If (required) changes are requested, the PR author should re-request a review from the reviewer once the comments have been addressed.
+
+### When your pull request is approved
+If you **are not a member of the Neurobagel developer team**: the reviewing Neurobagel maintainer is responsible for merging the PR.
+
+If you **are a member of the Neurobagel developer team** (i.e., you have write access to the repository): the PR author is responsible for merging the PR.
+
+## Have a question about contributing?
+At any point during a contribution, 
+please do not hesitate to [mention](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams) one of the core maintainers if you have a question or need further guidance, 
```

Comment:
I think it's a good idea to link the team page on at least of the `core maintainers`.